web/index.html

```diff
@@ -11,3 +11,9 @@
     <script type="module" src="/src/main.tsx"></script>
   </body>
 </html>
+<script>
+  // https://github.com/aws-amplify/amplify-js/issues/678
+  if (global === undefined) {
+    var global = window;
+  }
```

Comment:
Just a suggestion, what you have written is totally fine.
```suggestion
var global = global ?? window;
```